shared/domain/src/main/java/gov/va/vro/model/bgs/BgsApiClientRequest.java

```diff
@@ -0,0 +1,24 @@
+package gov.va.vro.model.bgs;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@RequiredArgsConstructor
+public class BgsApiClientRequest {
+
+  final String vbmsClaimId;
```

Comment:
might want to comment what this ties to as i know you want to rename the other files to match this but they aren't there yet.